app/controllers/cardsets_controller.rb

```diff
@@ -31,6 +31,7 @@ def create
 
   def index
     @cardsets = CardSet.all
+    @cardsets = CardSet.page(params[:page]).per(10)
```

Comment:
I don't think we actually want pagination in the Cardsets Controller. We're actually okay with getting everything.